Web/Edubase.Web.UI/Views/Shared/EditorTemplates/GovernorViewModel.cshtml

```diff
@@ -158,19 +154,17 @@ else
         <div class="govuk-form-group @Html.ValidationCssClassFor(x => x.TelephoneNumber)">
             <label class="govuk-label" for="GovernorTelephoneNumberInput">
```

Comment:
for needs updating to match the id of the label it is for - just TelephoneNumber here